package.json

```diff
@@ -29,9 +28,7 @@
     "yosay": "^2.0.2"
   },
   "devDependencies": {
-    "@oclif/dev-cli": "^1.21.0",
-    "@oclif/plugin-legacy": "^1.1.4",
```

Comment:
Heya @RasPhilCo curious on your opinion on removing `plugin-legacy`. I'm not _really_ sure if it's _needed_ or not but thought it would be a good opportunity to remove with oclif/core stuff?

Reply:
Cool!

Maybe this PR should be a few separate ones? test changes, linting changes, package changes